### PR TITLE
[Thumbnails] Fallback if clipping by embedded clipping path does not work

### PR DESCRIPTION
**Pull request by BlackbitNeueMedien**
When there is a clipping path embedded in a JPG it is tried to clip the image by this path. When this goes wrong for whatever reason, you get an error with the default placeholder image. 
The reason in our case was that ImageMagick did not support SVG: `ImagickException: no decode delegate for this image format "SVG"`

This PR adds a fallback that if clipping goes wrong for whatever reason, then the unclipped image will get delivered.
This is especially useful because `clip_auto_support` is enabled by default and when upgrading Pimcore versions some image thumbnails might suddenly break.

### CODE REVIEW

**Comment by NiklasBr** [comment]
Seems nice, we solved it by adding a custom tree preview configuration with preserve metadata enabled:

```yml
pimcore:
    assets:
        preview_image_thumbnail: treePreviewThumbnail
```

File: lib/Image/Adapter/Imagick.php [hunk 1]
```diff
@@ -151,12 +151,16 @@ public function load($imagePath, $options = [])
...
-                    $i->clipImage();
+                    try {
+                        // known issues:
+                        // - it seems that -clip doesnt work with the ImageMagick version
+                        //   ImageMagick 6.9.7-4 Q16 x86_64 20170114 (which is used in Debian 9)
+                        // - Imagick 3.4.4 with ImageMagick 7 on OSX has horrible broken clipping support
+                        $i->setImageAlphaChannel(\Imagick::ALPHACHANNEL_TRANSPARENT);
+                        $i->clipImage();
+                    } catch (\Exception $e) {
+                        Logger::info('Although automatic clipping support is enabled, your current ImageMagick / Imagick version does not support this operation');
```
Comment (brusch) [inline lib/Image/Adapter/Imagick.php:162]:
```suggestion
                        Logger::info(sprintf('Although automatic clipping support is enabled, your current ImageMagick / Imagick version does not support this operation on the image %s', $imagePath));
```

**Comment by brusch** [comment]
Changed the base to `6.8` as I'd classify this as a bug as it potentially could break existing behavior.

**Comment by BlackbitNeueMedien** [comment]
@brusch I am fine with your change. Do I have to do anything more?

**Comment by brusch** [comment]
No, just waiting for the tests to be finished 😉
Thanks a lot!

**Pull request merged**